spec/src/main/asciidoc/rest-endpoints.adoc

```diff
@@ -451,6 +466,38 @@ Counters do not have a suffix for the unit.
 application_visitors_total 80
 ----
 
+==== Concurrent Gauge OpenMetrics Text Format
+
+The values of the concurrent gauge must be the value of `getCount() / getMin() / getMax()`.
+The exposed metric names have suffixes as follows:
+
+|===
+| Value | Suffix
+
+| `getCount` |  `_total`
+| `getMin` |  `_min`
+| `getMax` |  `_max`
+|===
+
+
+The `_total` suffix is not appended if the (translated) original metric name already ends in `_total`.
+
+Concurrent gauges do not have a suffix for the unit.
+
+.Example OpenMetrics text format for a Concurrent Gauge
+[source, ruby]
+----
+# TYPE application_method_a_invocations_current gauge
```

Comment:
concurrentGauge instead of gauge?

Reply:
No, this is `gauge` as an OpenMetrics type (https://prometheus.io/docs/concepts/metric_types/#gauge), OpenMetrics doesn't have a `concurrentgauge` type